examples/plot_h2_grid.py

```diff
@@ -0,0 +1,47 @@
+"""
+Plot a grid on H2
+with Poincare Disk visualization.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from geomstats.hyperbolic_space import HyperbolicSpace
+import geomstats.visualization as visualization
+
+H2 = HyperbolicSpace(dimension=2)
+METRIC = H2.metric
+
+c = H2.intrinsic_to_extrinsic_coords
```

Comment:
maybe find a better (longer, more unique) name for the function if possible? it's easier for characters' replacements using regex in vim.